backend/dataall/modules/omics/__init__.py

```diff
@@ -0,0 +1,42 @@
+"""Contains the code related to X"""
+
+import logging
+from typing import Set, List, Type
+
+from dataall.base.loader import ImportMode, ModuleInterface
+from dataall.modules.omics.db.omics_repository import OmicsRepository
+
+log = logging.getLogger(__name__)
+
+
+class OmicsApiModuleInterface(ModuleInterface):
+    """Implements ModuleInterface for omics GraphQl lambda"""
+
+    @staticmethod
+    def is_supported(modes: Set[ImportMode]) -> bool:
+        return ImportMode.API in modes
+
+    @staticmethod
+    def depends_on() -> List[Type['ModuleInterface']]:
+        from dataall.modules.s3_datasets import DatasetApiModuleInterface
+
+        return [DatasetApiModuleInterface]
+
+    def __init__(self):
+        import dataall.modules.omics.api
+        from dataall.modules.omics.services.omics_permissions import GET_OMICS_RUN, UPDATE_OMICS_RUN
```

Comment:
I don't think we need to import permissions here. Line 27 is not needed anywhere